e2e/test/scenarios/dashboard/visualizer/basics.cy.spec.ts

```diff
@@ -451,4 +456,32 @@ describe("scenarios > dashboard > visualizer > basics", () => {
       cy.findByText("Created At: Month").should("exist");
     });
   });
+
+  it("should work with more than two datasets (VIZ-693)", () => {
```

Comment:
Should we move this to the cartesian test suite?

Reply:
Sure, why not